esptfa_arima/Authentication/serializers.py

```
@@ -47,14 +92,11 @@ class StudentSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Student
-        fields = '__all__'
+        fields = "__all__"
         depth = 1
 
     def validate(self, attrs):
         # validate the lrn to be only 11 chars long
-        if attrs['lrn'] is not None and len(attrs['lrn']) != 11:
+        if attrs["lrn"] is not None and len(attrs["lrn"]) != 11:
             raise serializers.ValidationError("LRN must be 11 characters long.")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)